### ISSUE TITLE
Print Validation Accuracy

### ISSUE THREAD

**Issue by herleeyandi** [rdcolema/tensorflow-image-classification#6]
Hello I just got this error, anybody can fix this?
![image](https://cloud.githubusercontent.com/assets/2309538/26757816/7bf7dd9e-48fc-11e7-8227-e75db9b988cd.png)

**Comment by rdcolema** [comment]
Hey Herleeyandi, can you take a look at #2 and see if that helps?

**Comment by salmiadu** [comment]
Hi rdcolema, i fassed the probelm of couldn't load the test data, i checked the dataset.py, i find everything fine. 
i think the issue is : the code couldn't load test images from the file (i had 1000 images in the test file)
![issue3](https://user-images.githubusercontent.com/20525376/27207063-1303befe-523c-11e7-82aa-feb35ded1fb0.JPG)
![issue2](https://user-images.githubusercontent.com/20525376/27207066-131220fc-523c-11e7-9a7c-0aad96b3c30b.JPG)
![issue1](https://user-images.githubusercontent.com/20525376/27207064-1309d8e8-523c-11e7-9555-ab87b603d5e5.JPG)

**Comment by salmiadu** [comment]
even if when i don't use jupyter, it's always Test-set 0 
![k](https://user-images.githubusercontent.com/20525376/27207285-f1a7057a-523d-11e7-84b4-d5c83b147011.JPG)
![esqdq](https://user-images.githubusercontent.com/20525376/27207289-f4f69e70-523d-11e7-8e8f-a72a55bd5ad2.JPG)
thank you Sir 👍
